core/src/main/java/tc/oc/pgm/tablist/MatchTabManager.java

```diff
@@ -318,4 +319,10 @@ public void onPlayerNameChange(NameDecorationChangeEvent event) {
     TabEntry entry = getPlayerEntryOrNull(Bukkit.getPlayer(event.getUUID()));
     if (entry instanceof DynamicTabEntry) ((DynamicTabEntry) entry).invalidate();
   }
+
+  @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+  public void onForceTabRefresh(RefreshPlayerTabEntryEvent event) {
```

Comment:
Consider we already have `NameDecorationChangeEvent` this seems to be very similar to this, could just reuse the same one?